page_source.html

```diff
@@ -0,0 +1 @@
+{'version': 'unknown', 'label': 'development', 'simulator': 'shieldhit'}
```

Comment:
what is that file ?

Reply:
it was generated while i was debugging selenium tests. I will delete it